packages/common/src/services/integrations/cleeng/CleengAccountService.ts

```diff
@@ -71,16 +77,29 @@ export default class CleengAccountService extends AccountService {
     return decodedToken.customerId;
   };
 
-  private getCustomer: GetCustomer = async (payload, sandbox) => {
-    return this.cleengService.get(sandbox, `/customers/${payload.customerId}`, { authenticate: true });
+  private getCustomer: GetCustomer = async (payload) => {
+    return this.cleengService.get(`/customers/${payload.customerId}`, { authenticate: true });
   };
 
-  private getLocales: GetLocales = async (sandbox) => {
-    return this.cleengService.getLocales(sandbox);
+  private getLocales: GetLocales = async () => {
+    return this.cleengService.getLocales();
   };
 
   initialize = async (config: Config, _url: string, logoutCallback: () => Promise<void>) => {
-    await this.cleengService.initialize(!!config.integrations.cleeng?.useSandbox, logoutCallback);
+    const cleengConfig = config?.integrations?.cleeng;
+
+    if (!cleengConfig?.id) {
+      throw new Error('Failed to initialize Cleeng integration. The publisherId is missing.');
+    }
+
+    // set accessModel and publisherId
+    this.publisherId = cleengConfig.id;
+    this.accessModel = cleengConfig.monthlyOffer || cleengConfig.yearlyOffer ? ACCESS_MODEL.SVOD : ACCESS_MODEL.AUTHVOD;
+    this.svodOfferIds = [cleengConfig?.monthlyOffer, cleengConfig?.yearlyOffer].filter(Boolean).map(String);
```

Comment:
Are we using this somewhere?

Reply:
`svodOfferIds` is the replacement for the `ConfigStore.getState().offers` property. It is used here:

(large diff)

https://github.com/Videodock/ott-web-app/pull/17/files#diff-8de786bf702d7d1e24bc671e13950f121f33d625116f89b0bdd700dd9b429f82R45-R47

https://github.com/Videodock/ott-web-app/pull/17/files#diff-2cc8f51c33e31e72515b60d3540281596786e5da7433a41e381d8f8f9b1a9592R22

Reply:
I still only see setting, not getting. :-p